core/src/test/scala/org/bykn/bosatsu/EvaluationTest.scala

```diff
@@ -3054,8 +3061,7 @@ test = Assertion(True, "")
 package A
 
 # this is basically a typecheck only
-x = (1, "1")
-(_, _) = x
```

Comment:
This shouldn't be allowed. In a pure language a total pattern match with no binding doesn't do anything.

The source converter should give an error in this case.